### PR TITLE
Connected actions to the cell.

### PR DESCRIPTION
**Pull request by roberthein**
Connected actions to the cell.

### CODE REVIEW

File: CounterExample/CounterCell.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//  CounterExample
+//
+//  Created by Katie Bogdanska on 11/11/16.
+//  Copyright © 2016 Colin Eberhardt. All rights reserved.
+//
+
+import UIKit
+import ReSwift
+
+typealias CellAction = (Bool) -> ()
```
Comment (roberthein) [inline CounterExample/CounterCell.swift:12]: This could be an enum to provide more context.

File: CounterExample/State/AppState.swift [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import ReSwift
 
+typealias Counter = Int
+
 struct AppState: StateType {
-    var counter: Int = 0
+    var counters: [Counter] = [Counter(), Counter(), Counter()]
```
Comment (roberthein) [inline CounterExample/State/AppState.swift:6]: replace initial setup with ability to add new counters

**Comment by roberthein** [comment]
Opened against the wrong fork. Will follow up.

**Comment by ColinEberhardt** [comment]
Aha - I guess @ashfurrow is running his workshop on this example. Have fun!

**Pull request closed without merging**